apps/pie-docs/src/components/text-input/overview/overview.md

```diff
@@ -0,0 +1,492 @@
+      src: "../../../assets/img/components/text-input/modifiers-alphanumeric-both.svg",
+      width: "256px",
+      alt: "Text input with alphanumeric content and both leading and trailing icons."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+#### Payment method
+
+Payment method is only available in trailing content.
+Use when a text input requires a card number to be entered, and reflect the card  entered by the user.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/modifiers-paymentmethod.svg",
+    alt: "Text input with payment method modifier.",
+    width: "256px"
+} %}
+
+### Type
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Placeholder</h4>
+    <p>Placeholder text should not contain crucial information, and only displayed if beneficial to the user.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/modifiers-type-placeholder.svg",
+      width: "256px",
+      alt: "Text input with placeholder type modifier."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Filled</h4>
+    <p>Placeholder text disappears after the user begins entering data into the input.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/modifiers-type-filled.svg",
+      width: "256px",
+      alt: "Text input with filled type modifier."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Sizes
+
+### Height
+
+Use a consistent text input height when it is being used alongside other form components on the same page.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Small</h4>
+    <p>Use when there isn’t enough vertical space for the default text input size.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/sizes-height-small.svg",
+      width: "256px",
+      alt: "Text input with small height."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Medium</h4>
+    <p>Default size and should be used whenever possible.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/sizes-height-medium.svg",
+      width: "256px",
+      alt: "Text input with medium height."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Large</h4>
+    <p>Choose this size when there is a lot of space to work with. This size is typically used in simple forms or when a text input is placed by itself on a page.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/sizes-height-large.svg",
+      width: "256px",
+      alt: "Text input with large height."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Width
+
+Use the field width to set the users’ expectations, this can help us communicate how much input is required. For example, if a field should only have 5 characters, a full width field would not be suitable.
+When used in a form, all text input widths should be aligned.
```

Comment:
Is this phrasing accurate? Something like a payment form would be an exception, perhaps, with the card security code being shorter than the card number.

Reply:
(Happy for this to be addressed in a follow-up)

Reply:
@RLumleyJET could you help, please? Is this phrasing accurate?

Reply:
Use the field width to set the users’ expectations, this can help us communicate how much input is required. For example, if a field should only have 5 characters, a full width field would not be suitable.
In forms, aim to standardise widths for text inputs to enhance readability for users.

an adjustment from Rachel